front/components/assistant_builder/actions/ProcessAction.tsx

```diff
@@ -48,11 +49,26 @@ export function isActionProcessValid(
       return false;
     }
   }
-  return (
-    action.type === "PROCESS" &&
-    Object.keys(action.configuration.dataSourceConfigurations).length > 0 &&
-    !!action.configuration.timeFrame.value
-  );
+  if (Object.keys(action.configuration.dataSourceConfigurations).length === 0) {
+    return false;
+  }
+  if (!action.configuration.timeFrame.value) {
+    return false;
+  }
+  if (
+    action.configuration.tagsFilter &&
+    action.configuration.tagsFilter.in.filter((tag) => tag === "").length > 0
```

Comment:
nit: would be slightly more readable as `action.configuration.tagsFilter.in.some(...)`

Reply:
sure will update :+1: